src/ordering.c

```diff
@@ -193,6 +193,7 @@ const char *get_section(const struct policy_node *node)
 	case NODE_RT_RULE:
 		return "_non_ordered";
 	case NODE_ROLE_ALLOW:
+	case NODE_ROLE_TYPES:
 		// This is not in the style guide. I normally see it grouped
```

Comment:
I think comment is true for both.  Can you change it to a "these are" and "them" now that it refers to two items?

Reply:
updated comment